lib/modules/wdpa/overseas_territories_importer.rb

```diff
@@ -0,0 +1,30 @@
+module Wdpa::OverseasTerritoriesImporter
+  OVERSEAS_TERRITORIES_CSV = "#{Rails.root}/lib/data/seeds/overseas_territories.csv".freeze
+  extend self
+
+  def import
+    ActiveRecord::Base.transaction do
+      csv = CSV.read(OVERSEAS_TERRITORIES_CSV)
+      csv.shift # remove headers
+
+      csv.each do |parent_iso, child_isos|
+        parent_country = Country.find_by_iso_3(parent_iso)
+        child_isos = child_isos.split(';')
+
+        unless parent_country.nil?
```

Comment:
you can do a `next if parent_country.nil?` here, and save an indentation level 😀